test/systemtest/config.py

```diff
@@ -102,4 +102,12 @@
         "name": "dif_gpio_smoketest",
         "targets": ["fpga_nexysvideo"],
     },
+    {
```

Comment:
Please add these tests to `hw/top_earlgrey/dv/verilator_sim_cfg.hjson` as well.

Reply:
Done.  Could you check that the syntax is correct and/or advise me on how to invoke `dvsim` to either spawn the tests or check the syntax of the config?